### PR TITLE
Add TIaaS post

### PR DESCRIPTION
**Pull request by beatrizserrano**
The authors don't appear in the local rendered version, don't know why

### CODE REVIEW

File: _posts/2022-04-28-tiaas-hans-rudolf-lucille.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
+title: 'Training Infrastructure Feedback from Hans-Rudolf Hotz and Lucille Delisle'
+tags: [training, TIaaS]
+supporters:
+- galaxy-europe
+author: [hrhotz, lldelisle]
```
Comment (bgruening) [inline _posts/2022-04-28-tiaas-hans-rudolf-lucille.md:7]: not sure if this will work. if this does not map to `peoples` it should maybe be a full name?

Comment (beatrizserrano) [reply _posts/2022-04-28-tiaas-hans-rudolf-lucille.md:7]: can only authors in the people.yml file be added with their IDs?

Comment (bgruening) [reply _posts/2022-04-28-tiaas-hans-rudolf-lucille.md:7]: I think so, not sure.

**Comment by hrhotz** [comment]
just curious: what about a link to https://galaxyproject.org/people/hansrudolf-hotz/ ?

**Comment by beatrizserrano** [comment]
> just curious: what about a link to https://galaxyproject.org/people/hansrudolf-hotz/ ?

We could use that link in your signature for instance, but it wouldn't show your avatar like in [this example](https://galaxyproject.eu/posts/2022/04/20/towards-sustainable-storage/), that's what I wanted to get. In my dream world where the content is migrated to the Hub, and we have a Community Directory, something like that would work I guess.

**Pull request merged**